server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/LogicalDatabaseChangeController.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.server.web.controller.v2;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.oceanbase.odc.service.common.response.Responses;
+import com.oceanbase.odc.service.common.response.SuccessResponse;
+import com.oceanbase.odc.service.connection.logicaldatabase.LogicalDatabaseChangeService;
+import com.oceanbase.odc.service.connection.logicaldatabase.model.SqlExecutionUnitResp;
+import com.oceanbase.odc.service.task.exception.JobException;
+
+/**
+ * @Author: Lebie
+ * @Date: 2024/9/4 12:20
+ * @Description: []
+ */
+@RestController
+@RequestMapping("/api/v2/logicaldatabase")
+public class LogicalDatabaseChangeController {
+    @Autowired
+    private LogicalDatabaseChangeService logicalDatabaseChangeService;
+
+    @RequestMapping(value = "/scheduleTasks/{scheduleTaskId:[\\d]+}/executionUnits/{executionUnitId:[\\d]+}",
+            method = RequestMethod.GET)
+    public SuccessResponse<SqlExecutionUnitResp> detailPhysicalDatabaseChangeTask(@PathVariable Long scheduleTaskId,
+            @PathVariable Long executionUnitId) {
+        return Responses.success(logicalDatabaseChangeService.detail(scheduleTaskId, executionUnitId));
+    }
+
+    @RequestMapping(
+            value = "/scheduleTasks/{scheduleTaskId:[\\d]+}/executionUnits/{executionUnitId:[\\d]+}/skipCurrent",
```

Comment:
skipCurrent-->skip, will it keep same meaning ?

Reply:
No, it's not actually the same meaning. 
The `executionUnit` in the URL actually represents a physical database execution unit, and each SQL execution record under the physical database has not been abstracted into a separate resource, so the front end is not aware of it. `skipCurrent` indicates skipping the execution of the current executing SQL, whereas `skip` would indicate skipping the execution of the entire physical database.

Reply:
done